cmd/evaluate_all_command.go

```diff
@@ -75,7 +75,12 @@ func evaluateAll(cmd *cobra.Command, args []string) (cmdError error) {
 		return err
 	}
 
-	decoder, err := configureDecoder(true)
+	inputFilename := ""
+	if len(args) > 0 {
+		inputFilename = args[0]
+	}
+
+	decoder, err := configureDecoder(false, inputFilename)
```

Comment:
found this issue - this has been set to false!

Reply:
Ahh, clearly a copy-paste mistake, fixed and pushed 😂